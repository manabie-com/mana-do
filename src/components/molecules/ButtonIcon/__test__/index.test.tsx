import React from 'react';
import {render} from '@testing-library/react';
import {ButtonIcon} from "../index";

describe('<ButtonIcon /> render', () => {
    it('should match snapshot', function () {
        const Element = render(<ButtonIcon iconName="star" onClick={jest.fn()}/>);
        expect(Element.container.firstChild).toMatchSnapshot();
    });
})