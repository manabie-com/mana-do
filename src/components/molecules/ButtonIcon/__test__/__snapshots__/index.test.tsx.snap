// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<ButtonIcon /> render should match snapshot 1`] = `
<div
  class="btn"
>
  <i
    class="manabie-star"
  />
  <span />
</div>
`;
